packages/runtime/src/index.js

```
@@ -0,0 +1,47 @@
+import * as esJscore from 'https://esm.run/@es-js/core';
+import * as esJseshtml from 'https://esm.run/@es-js/eshtml';
+window.addEventListener("load", function(){
+  let etiquetas = document.getElementsByTagName('script');
+  for(var i = 0; i < etiquetas.length; i++){
```

Biome

[error] This var should be declared at the root of the enclosing function.

The var is accessible in the whole body of the enclosing function.
To avoid confusion, it should be declared at the root of the enclosing function.

(lint/correctness/noInnerDeclarations)